works.yaml

```diff
@@ -2903,8 +2872,7 @@ Abaco:
   license:
   - Lucent Public License
   license_reference:
-  - http://plan9.bell-labs.com/plan9/license.html
-  - http://plan9.bell-labs.com/sources/plan9/sys/src/cmd/abaco/
+  - []
```

Comment:
When using the [ ] syntax, it should be on the same line as the `license_reference` key, not in a separate-line list item.